### ISSUE TITLE
An error on babel6

### ISSUE THREAD

**Issue by zakuro9715** [azu/espower-babel#21]
I found an error on babel6 project. Please see the example project.

https://github.com/zakuro9715/espower-babel-babel6-error-example

On babel5

```
$ npm run test

> @ test /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel5
> mocha --compilers js:espower-babel/guess test/add.test.js



  add
    ✓ should returns result of A plus B


  1 passing (8ms)
```

On babel6

```
$ npm run test

> @ test /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6
> mocha --compilers js:espower-babel/guess test/add.test.js

/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/logger.js:58
    throw new Constructor(this._buildMessage(msg));
    ^

ReferenceError: [BABEL] /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/test/add.test.js: Unknown option: direct.presets
    at Logger.error (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
    at OptionManager.mergeOptions (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:29)
    at OptionManager.init (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/options/option-manager.js:216:10)
    at File.initOptions (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/index.js:147:75)
    at new File (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/file/index.js:137:22)
    at Pipeline.transform (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at Object.extensions.(anonymous function) [as .js] (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/espower-babel/index.js:56:28)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/lib/mocha.js:216:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/lib/mocha.js:213:14)
    at Mocha.run (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/lib/mocha.js:453:10)
    at Object.<anonymous> (/home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/node_modules/mocha/bin/_mocha:393:18)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3

npm ERR! Linux 4.2.0-1-amd64
npm ERR! argv "/home/zakuro/.nvm/versions/node/v5.0.0/bin/node" "/home/zakuro/.nvm/versions/node/v5.0.0/bin/npm" "run" "test"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE
npm ERR! @ test: `mocha --compilers js:espower-babel/guess test/add.test.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ test script 'mocha --compilers js:espower-babel/guess test/add.test.js'.
npm ERR! This is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mocha --compilers js:espower-babel/guess test/add.test.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/zakuro/src/github.com/zakuro9715/espower-babel-babel6-error-example/babel6/npm-debug.log
```

**Comment by azu** [comment]
> Unknown option: direct.presets

Direct reason is here(`presets` is introduced since Babel 6).
espower-babel doesn't support Babel 6 yet.

espower-babel wait for [babel-plugin-espower](https://github.com/power-assert-js/babel-plugin-espower/pull/5).

**Comment by twada** [comment]
Hi, @zakuro9715 

Babel 6 is not stable enough so I do not recommend you to use babel6.

However if you want to use babel6 now, please use [babel6 branch](https://github.com/power-assert-js/babel-plugin-espower/pull/5) of babel-plugin-espower directly.

``` json
  "devDependencies": {
    "babel-plugin-espower": "power-assert-js/babel-plugin-espower#babel6",
  },
```

**Comment by zakuro9715** [comment]
@twada Thank you very much.

My team has decided to use babel5.

**Comment by twada** [comment]
FYI: I just released [babel-plugin-espower 2.0.0](https://github.com/power-assert-js/babel-plugin-espower/blob/master/CHANGELOG.md#200-2015-11-13). Now power-assert works with Babel6!

**Comment by azu** [comment]
babel-plugin-espower [v2.0.0](https://github.com/power-assert-js/babel-plugin-espower/releases/tag/v2.0.0) is released. it support Babel 6.

We'll update espower-babel too.

**Comment by twada** [comment]
@azu Great!!

**Comment by azu** [comment]
@zakuro9715 We just released espower-babel [4.0.0](https://github.com/power-assert-js/espower-babel/releases/tag/4.0.0).
It support Babel 6.
